apps/cowswap-frontend/src/modules/account/containers/AccountDetails/index.tsx

```diff
@@ -194,9 +198,7 @@ export function AccountDetails({
                 )}
               </WalletSelector>
 
-              {(ENSName || account) && (
-                <Copy toCopy={ENSName ? ENSName : account ? account : ''} />
-              )}
+              {(ENSName || account) && <Copy toCopy={ENSName ? ENSName : account ? account : ''} />}
```

Comment:
is this expression sth that could be expressed simpler? 

would it be equivalent to: `ENSName || account || ''`

Reply:
It might, will change it after this PR. I try to keep this PR as simple as possible, just imports replacements